### ISSUE TITLE
No negative values for inclination?

### ISSUE THREAD

**Issue by MorpheusFT** [KSP-KOS/KOS#1966]
nextnode:orbit:inclination

Ran into some problems with my scripts. nextnode:orbit:inclination doesnt seem to give any negative values.
I manually  changed a manoeuvre node from positive to negative inclination, then used print(nextnode:orbit:inclination); but both times it gives a positive value.

**Comment by ElWanderer** [comment]
Inclination is always positive. If you reverse an inclined orbit, the effect is not to negate the inclination, but to move around the ascending node by 180°.

**Comment by MorpheusFT** [comment]
Well, that makes sense, but not helpfull for my purposes.

Because my script finds the eta to ascending node, then tries to find the correct dV for the inclination (by way of hillclimbing, while keeping the node in the same spot), but since a positive inclination is returned both ways, it either doesn't do anything or gives mirrored inclination and only on rare occasion when all stars allign, the correct inclination.

Anyway Ill have to find a different solution then. (Ill keep this open for a day or so, then I will close it).

**Comment by ElWanderer** [comment]
If you're hillclimbing, you could take into account the LAN of the resultant orbit in your scoring function, penalising those nodes that change/flip the LAN unless you are aiming to turn the Ascending Node into the Descending Node (and noting that if you're trying to reach 0 inclination, the LAN is skittish/unreliable/best ignored).

A plane change is something that can be precisely calculated mathematically, but there are a variety of ways of doing it.

**Comment by MorpheusFT** [comment]
Yeah I tried both ways (including LAN and calculating the dV mathematically). 

But every time the resulting inclinination is about one degree of. Whether this is because I start at small non-zero inclination or because of a slightly eccentric orbit (throwing my dV calculation slightly off) I don't yet know. 

I tried to to have it do the two things over and over again. Adjusting location of node due to dV and then adjusting dV due to location of node and then the same again). Can't seem to get the error in inclination smaller than 1 degree. (the contract requires a specific LAN and inclination)
